src/util/desktop-notifier.js

```diff
@@ -0,0 +1,26 @@
+/* @flow */
+import {NotificationCenter} from 'node-notifier';
+
+const notifier = new NotificationCenter({
+  withFallback: true,
+});
+
+type desktopNotificationsParams = {
+  titleString: string,
```

Comment:
let's rename `notifierSource` to `notifier`, `titleString` to `title` and `messageString` to `message`, it will also help to make it line 22 shorter like in:
```
notifier.notify({title, message});
```